sample/lib/widgets/showcase/sample_input_phone.showcase.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/src/widgets/framework.dart';
+
+class SampleInputPhoneShowcase extends StatelessWidget {
```

Comment:
To make that file more generic and serve as a shell for all other inputs, you should remove the ```Phone``` keyword from the name of it.

E.g.:
```SampleInputShowcase```
```sample_input.showcase.dart```